querybook/webapp/components/QueryReviewsNavigator/QueryReviewItem.tsx

```diff
@@ -23,150 +26,129 @@ interface IQueryReviewItemProps {
+                    length={50}
+                    seeLess
+                />
             </div>
         </div>
     );
 };
 
-const ReviewTimestamp: React.FC<{
+interface MetaInfoProps {
     review: IQueryReview;
-    type: ReviewType;
-}> = ({ review, type }) => (
-    <AccentText
-        className="horizontal-space-between"
-        size="xsmall"
-        color="lightest"
-    >
-        <span>
-            {type === 'assigned' ? 'Requested' : 'Created'} at:{' '}
-            {generateFormattedDate(review.created_at)}
-        </span>
-    </AccentText>
+    visibleReviewers: number[];
+    extraReviewers: number;
+    userInfoById: IStoreState['user']['userInfoById'];
+}
+
+const MetaInfo: React.FC<MetaInfoProps> = ({
+    review,
+    visibleReviewers,
+    extraReviewers,
+    userInfoById,
+}) => (
+    <div className="meta-info">
+        <div className="reviewers">
+            <span aria-label="Reviewers" data-balloon-pos="up">
+                <Icon name="Users" size={14} className="mr4" />
+            </span>
+            <UserAvatarList
+                users={visibleReviewers.map((id) => {
+                    const userInfo = userInfoById[id];
+                    return {
+                        uid: id,
+                        tooltip: userInfo
+                            ? userInfo.fullname ?? userInfo.username
+                            : `User ${id}`,
+                    };
+                })}
+                extraCount={extraReviewers}
+            />
+        </div>
+        <div className="timestamp">
+            <span aria-label="Updated at" data-balloon-pos="up">
+                <Icon name="Calendar" size={14} className="mr4" />
+                <AccentText size="xsmall" color="light">
+                    {generateFormattedDate(review.created_at)}
+                </AccentText>
+            </span>
+        </div>
+    </div>
 );
 
 export const QueryReviewItem: React.FC<IQueryReviewItemProps> = ({
     review,
-    type,
     isSelected,
     onClick,
 }) => {
-    const selfRef = useRef<HTMLDivElement>();
-
+    const selfRef = useRef<HTMLDivElement>(null);
     const queryExecutionUrl = useMemo(
         () => `/query_execution/${review.query_execution_id}/`,
         [review.query_execution_id]
     );
-
     const handleClick = useCallback(() => {
         onClick?.();
         navigateWithinEnv(queryExecutionUrl);
     }, [queryExecutionUrl, onClick]);
 
-    const statusColor: Status =
-        STATUS_COLOR_MAP[review.status] ?? Status.warning;
-
-    const className = clsx('QueryReviewItem', {
-        'is-selected': isSelected,
-    });
+    const className = clsx('QueryReviewItem', { 'is-selected': isSelected });
+    const userInfoById = useSelector(
+        (state: IStoreState) => state.user.userInfoById
+    );
+    const visibleReviewers = review.reviewer_ids.slice(0, 3);
+    const extraReviewers = Math.max(0, review.reviewer_ids.length - 3);
```

Comment:
do this inside MetaInfo, these arent passed other places
also useMemo to prevent a new array generated per render cycle